src/app/Plans/PlansPage.tsx

```diff
@@ -33,22 +33,15 @@ const PlansPage: React.FunctionComponent = () => {
 
   const [isWizardOpen, toggleWizard] = React.useReducer((isWizardOpen) => !isWizardOpen, false);
 
-  let addPlanDisabledObj: IAddPlanDisabledObjModel = {
-    isAddPlanDisabled: false,
-    disabledText: 'Click to create a new migration plan',
+  let addPlanDisabledObj: Pick<IAddTooltipProps, 'isEnabled' | 'content'> = {
+    isEnabled: false,
+    content: '',
   };
 
-  if (vmwareList.length < 1) {
-    addPlanDisabledObj = {
-      isAddPlanDisabled: true,
-      disabledText: 'At least 1 VMware provider is required to create a plan.',
-    };
-  }
-
-  if (cnvList.length < 1) {
+  if (vmwareList.length < 1 || cnvList.length < 1) {
     addPlanDisabledObj = {
-      isAddPlanDisabled: true,
-      disabledText: 'At least 1 OpenShift provider is required to create a plan.',
+      isEnabled: true,
+      content: 'At least 1 source and 1 target providers are required to create a plan.',
```

Comment:
@vconzola this text will be used as a tooltip on the Create Plan button when it's disabled. What do you want it to say? Didn't see this in the mockup.

Reply:
@mturley First, I don't think we need any tooltip when the Create plan button is enabled - it's self-explanatory what it will do. When the button is disabled, I think it should say, "You must add at least VMware provider and one OpenShift Virtualization provider in order to create a migration plan." Will need to re-word when we have more sources than VMware, but we haven't surfaced the "source" and "target" terms up to this point in the workflow, so I don't want to introduce them here.

Reply:
@vconzola did you mean "at least one VMware provider" instead of "at least VMware provider"?

Reply:
@vconzola, please see the Tooltip message when no providers are available.

![plan-wizard-add-disabled](https://user-images.githubusercontent.com/1901741/93367925-78bc5480-f84d-11ea-8961-a59f3d2710aa.png)